### PR TITLE
fix(npm): allow to inspect npm modules with --inspect-brk

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/denoland/deno/issues/16757

### CODE REVIEW

File: runtime/worker.rs [hunk 1]
```diff
@@ -343,7 +349,7 @@ impl MainWorker {
     self.evaluate_module(id).await
   }
 
-  fn wait_for_inspector_session(&mut self) {
+  pub fn wait_for_inspector_session(&mut self) {
```
Comment (bartlomieju) [inline runtime/worker.rs:352]: Revert

File: runtime/web_worker.rs [hunk 2]
```diff
@@ -466,6 +466,12 @@ impl WebWorker {
       ..Default::default()
     });
 
+    {
```
Comment (bartlomieju) [inline runtime/web_worker.rs:469]: Add comment

File: runtime/worker.rs [hunk 3]
```diff
@@ -244,6 +244,12 @@ impl MainWorker {
       ..Default::default()
     });
 
+    {
```
Comment (bartlomieju) [inline runtime/worker.rs:247]: Add comment

File: runtime/worker.rs [hunk 4]
```diff
@@ -250,6 +250,12 @@ impl MainWorker {
         &mut js_runtime,
         options.should_break_on_first_statement,
       );
+
+      // Put inspecto handle into the op state so we can put a breakpoint when
```
Comment (dsherret) [inline runtime/worker.rs:254]:
```suggestion
      // Put inspector handle into the op state so we can put a breakpoint when
```

File: runtime/web_worker.rs [hunk 5]
```diff
@@ -472,6 +472,12 @@ impl WebWorker {
         &mut js_runtime,
         false,
       );
+
+      // Put inspecto handle into the op state so we can put a breakpoint when
```
Comment (dsherret) [inline runtime/web_worker.rs:476]:
```suggestion
      // Put inspector handle into the op state so we can put a breakpoint when
```

**Review by dsherret — APPROVED** [review]
LGTM

**Pull request merged**